libs/light/neopixel.ts

```diff
@@ -499,11 +499,11 @@ namespace light {
         }
 
         /**
-         * This function is deprecated.
+         * Sets the photon hue.
          */
-        //% blockId=neophoton_set_color block="%strip=variables_get|photon set pen hue %hue=colorWheelPicker"
+        //% blockId=neopixel_set_color block="%strip=variables_get|photon set pen hue %hue=colorWheelPicker"
         //% help="light/set-photon-pen-hue"
-        //% parts="neopixel" deprecated=1 blockHidden=true
```

Comment:
was this by design or did i mess up the merge? setPhotonPenHue is new and needs to stay.

Reply:
Looks like a merge issue. Will fix

Reply:
Just to clarify, you want this block deprecated correct?

Reply:
I think in this case, there's no need for this as we will attach the neophoton_set_color block to the __setPhotonColor function and get it to call setPhotonPenColor.

Reply:
No, photon needs hue - it does not work well with color. So setPhotonPenHue is the block that is used.

Reply:
Okay, but you set the block as deprecated and hidden. Did you mean to do that? 
Are you wanting to ship two blocks, one for pen color, one for pen hue?

Reply:
Let's see: ``setPhotonColor`` is the old API with a confusing name which stays for legacy. ``setPhotonPenHue`` is the one that's useful so that's the one to add. The ``setPhotonPenColor`` one, I'm not sure - in doubt we can remove it.

Reply:
Okay so we need to remove the deprecated flag on the setPhotonPenHue and give it a new ID that does not need the __ hack.

Reply:
Fixed, please verify